examples/basic/line/script.js

```diff
@@ -0,0 +1,150 @@
+import * as UTILS from '../../globals';
+
+const controlsModule = new WHS.controls.OrbitModule();
+
+const world = new WHS.App([
+  ...UTILS.appModules({
+    position: new THREE.Vector3(0, 10, 200)
+  }),
+  controlsModule
+]);
+
+controlsModule.controls.autoRotate = true;
+
+const yPadding = 2;
+const xzPadding = 1;
+let yDist = Math.floor(Math.random() * 10) + yPadding;
+for (let i = 1; i < 100; i++) {
+  const xStart = Math.floor(Math.random() * i * (i % 2 === 0 ? -1 : -1)) + (xzPadding * (i % 2 === 0 ? -10 : 10));
+  const xDist = Math.floor(Math.random()) + 5;
+  const zStart = Math.floor(Math.random() * i * (i % 2 === 0 ? -1 : -1)) + (xzPadding * (i % 2 === 0 ? -10 : 10));
+  yDist = Math.floor(Math.random() * 50) + (yPadding * 10);
+  const bottomPoint1 = {x: xStart, y: 0, z: zStart};
+  const bottomPoint2 = {x: xStart + xDist, y: 0, z: zStart};
+  const bottomPoint3 = {x: xStart, y: 0, z: -xDist + zStart};
+  const bottomPoint4 = {x: xStart + xDist, y: 0, z: -xDist + zStart};
```

Comment:
I don't like such `bottomPoint1`, `bottompoint2` and so on... 

You can just make an array of `THREE.Vector3`